### ISSUE TITLE
Static Folder Serving

### ISSUE THREAD

**Issue by jetdevuk** [jetdevuk/sylph#5]
Sylph needs to be able to serve static assets on certain routes. Currently the /server/public folder is totally useless.

**Comment by jetdevuk** [comment]
Nanoexpress is meant to have this but there is no sign of the method in the code and following the docs causes an error.

**Comment by jetdevuk** [comment]
Thought resolved but using:

```js
app.use(require('nanoexpress/build/static.js')(publicFolder));
```

Breaks everything, it tries to static serve every endpoint by default instead of in a fallback pattern.

**Comment by jetdevuk** [comment]
Fixed in Sifrr Migration (1.5.0)
